src/any_agent/frameworks/langchain.py

```diff
@@ -110,18 +454,20 @@ async def _run_async(self, prompt: str, **kwargs: Any) -> str | BaseModel:
                 structured_output_message,
             ]
 
-            # Use response schema if supported by the model
-            if supports_response_schema(model=self.config.model_id):
-                completion_params["response_format"] = self.config.output_type
```

Comment:
WIth this behavior change, we would now enforce that if you are asking for a response schema, the provider/model has to support returning a response schema.